_data/gaming.yml

```diff
@@ -84,6 +84,14 @@ websites:
       - totp
     doc: https://www.epicgames.com/help/en-US/c74/c112/a3218
 
+  - name: ESL
+    url: https://account.eslgaming.com
+    img: esl.png
+    tfa:
+      - totp
+      - u2f
+    doc: https://account.eslgaming.com/mfa
```

Comment:
Please change the doc url to something that doesn't require the user to sign in.

Reply:
We currently don't have a public doc url. Shall I remove the doc link?

Reply:
If no documentation is available could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
![screenshot](https://user-images.githubusercontent.com/3535780/62257740-32b4ef00-b407-11e9-8f35-fc7c47ad72fb.gif)

Reply:
* Setup MFA:
![Screenshot from 2020-09-02 20-40-19](https://user-images.githubusercontent.com/1951610/92023160-a8069800-ed5c-11ea-8edb-7b1a49cd894d.png)
* Login with TOTP
![Screenshot from 2020-09-02 20-36-12](https://user-images.githubusercontent.com/1951610/92023163-a937c500-ed5c-11ea-8423-02ffe3ca3007.png)
* Login with security key (webauthn)
![Screenshot from 2020-09-02 20-35-39](https://user-images.githubusercontent.com/1951610/92023164-a937c500-ed5c-11ea-991b-cc0bf52542c6.jpg)

Reply:
Thanks! 😄